### DISCUSSION TITLE
Static linking?

### DISCUSSION THREAD

**Discussion by Barugon** [georust/gdal#436] [Q&A] [answered]
Is it possible to statically link GDAL?

**Comment by lnicola** [comment, marked as answer]
See https://github.com/georust/gdal/tree/master/gdal-sys. You can try `GDAL_STATIC=1`, but at least on my system (Fedora) it doesn't work because `unixODBC` is missing a static library. I didn't try the official container, though.

**Reply by Barugon** [reply]
Thanks, I'll give it a go.
